INSTALL-linux.md

```diff
@@ -0,0 +1,177 @@
+# Installing the CircleCI Runner on Linux
+
+## Prerequisites
+
+### Installation Tooling
+
+The installation process assumes that the following utilities are installed on the system:
+
+* curl
+* sha256sum (installed as part of `coreutils`)
+* systemd
+
+Installation also requires permissions to create a user, and create directories under `/opt`.
```

Comment:
These prereqs are required for the download too I think, so putting them here is too late?